src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/controllers/web/WorkflowRunControllerTests.kt

```diff
@@ -208,6 +216,16 @@ class WorkflowRunControllerTests
             assertThat(it.instances).isEqualTo(workflowRunRequest.instances)
             assertThat(it.gitBranch).isEqualTo(workflowRunRequest.gitBranch)
             assertThat(it.gitCommit).isEqualTo(workflowRunRequest.gitCommit)
+            assertThat(it.workflowRunReports).isEqualTo(
```

Comment:
Again, I think the changes you've made have actually highlighted a weakness of the existing test, which is that, although the workflow run request contains 2 reports, the `mockAPIResponseText` only returns one key - it would be nice to modify that to return two keys, so that we can check the zip of two arrays of same length (as would be expected) works.